tests/foreman/endtoend/test_api_endtoend.py

```diff
@@ -1140,34 +1136,33 @@ def test_positive_end_to_end(self, fake_manifest_is_set, default_sat, rhel6_cont
         repositories = []
 
         # step 2.5: Create custom YUM repository
-        repo1 = entities.Repository(
+        yum_repo = entities.Repository(
```

Comment:
```suggestion
        custom_repo = entities.Repository(
```

Optional, non-blocking comment!

Reply:
Updated.